### DISCUSSION TITLE
Como vocês separam trabalho e estudos? Ou não separam at all?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#406] [Discussão]

**Comment by elvisdetona** [comment]
Eu estudo (ingles e algo novo de tecnologia quando precisa) apenas no trabalho. 

Primeiro horário eu faço aulas de inglês, e quando "estou sussa" ou "quando preciso", eu paro e estudo algo novo (ultimas coisas que estudei foram relacionadas a javascript / react) .

**Comment by hjdesigner** [comment]
Estudo inglês no percurso do trabalho até o serviço e vice e versa.
E tento estudar pelo menos 1h toda noite coisas de Front-End.
*Lógico quando meu filho deixa rsrs

**Comment by felipebernardes** [comment]
O que dá certo pra mim é incorporar aprendizado em atividades leves do dia-a-dia:

Pra inglês, 
- vejo séries/filmes SEMPRE sem legenda ou com legenda em inglês quando tô com preguiça 
- tenho dois amigos com os quais só falo em inglês no whatsapp (no bar a gente fala em pt)
- me divirto lendo [reddit](reddit.com) e [hackernews](https://news.ycombinator.com/)

Pra código, 
- novidades: toda semana eu dou uma olhada nas newsletters do CodePen/[BrazilJS](https://braziljs.org/#weekly)/[PonyFoo](https://ponyfoo.com/weekly)
- prática: TENTO fazer um [pen](http://codepen.io/) por semana
- ensinando: [onde trabalho](http://www.tegra.com.br/), diariamente há uma mini talk de 10 minutos sobre conceitos técnicos pros estagiários. Toda quinta, é minha vez de dar a talk. Sempre reviso algum conceito por 15 ou 20 minutos antes de dar a talk 
- conceitos: reservo cinco~dez minutos quando chego no trabalho pra ler a lesson of the day do [Enki](https://www.enki.com/)

**Comment by bernardodiasc** [comment]
Simplificando, estudo durante o trabalho e mando a conta pro cliente. ;)

**Comment by k8adev** [comment]
Não separo (como sabemos, eu não tenho vida).

Com idiomas, como meus projetos são pt-br, sa/en, estudo e pratico meu inglês no decorrer do desenvolvimento deles, além de escrever commit, documentação e etc. Quando não sei como escrever algo, aproveito para dar aquela googada para aprender. Também tenho alguns plugins de idiomas que me auxiliam nessa prática.

Com tecnologias e afins, estudo conforme quero ou preciso usar, ou se quero ensinar algo pro meu time mas não sei como explicar bem. 
Não consigo estudar todas as coisas novas que saem e na real, admiro quem consegue, rs.

( 　ﾟ,_ゝﾟ)

**Comment by eduardojmatos** [comment]
Eu tenho estudado mais em casa, mas assuntos não totalmente relacionados ao trabalho. A não ser que tenhamos a proposta de fazer algo muito novo que ninguém tenha ideia de como fazer. Nesse caso eu estudo em casa sim coisas pro trampo.

Na empresa, dependendo do sprint, sempre fazemos alguma POC (prove of concept) pra alguma história, daí surge a oportunidade de estudar.

**Comment by malaquiasdev** [comment]
Eu tenho estudado em casa coisas não relacionadas ao trabalho (já tenho dominio sobre o que preciso utilizar para trabalhar) geralmente são tecnologias que me tiram um pouco da minha zona de conforto e que vão fazer a minha stack atual melhorar.

Por exemplo: Sou um desenvolvedor Java. Na parte do front-end costumo trabalhar com [JSF](https://javaserverfaces.java.net/), desejando melhorar minha stack atual e trabalhar mais com arquiteturas de backend REST, passei a estudar Angular 2.

Inglês não estudo mais, só prático mesmo lendo artigos da área ou coisas o mundo, ouvindo podcasts e assistindo videos sem legenda.

**Comment by suissa** [comment]
Pego trampo de coisas que NUNCA fiz para aprender coisas novas e assim unir o útil ao muito agradável, que é estudar.

Tudo que estudo fora programação consigo converter em exemplos com código, por exemplo: dialética, matemática, física, química, idiomas...

Tento criar uma ponte entre o abstrato e o concreto para fixar melhor os conceitos e com isso pratico a programação sem precisar ser trabalhando, também. :p

**Comment by thulioph** [comment]
Aqui no trabalho todos os clientes são de fora, pratico o inglês no slack, commits, documentações e lendo artigos, além de um curso que faço pela manhã.

Eu tento toda semana por em prática ao menos 1 (uma) coisa nova, seja ver um vídeo sobre testes e por em prática, seja ler um artigo e experimentar uma nova api, etc.. Nem sempre é possível, mas consigo manter um ritmo e não ficar parado.

Normalmente no percurso de casa -> trabalho / trabalho -> casa, eu vou lendo newsletters ou blogs, as vezes podcasts ou qualquer assunto relacionado a tecnologia, porém apenas leituras.

Nos finais de semana eu tento não ver código e todo tempo é com minha namorada.

**Comment by vitor-mendrone-deel** [comment]
Utilizo meu tempo livre pra estudar sempre que possível e no trabalho procuro aplicar o que estou aprendendo. 

Gosto de montar planos de estudo pra quando tenho períodos livres maiores e também mantenho uma lista de livros/artigos pra ler durante os dias úteis (Minha regra é ler pelo menos 5 livros técnicos por ano e outros 15 ou 20 de assuntos gerais)

Atualmente estou seguindo um plano de estudos de Javascript que montei a alguns meses focando nos pontos que eu via como mais obscuros pra mim. (Async/Concurrent programming) 

No trabalho só estudo coisas que surgirem de ultima hora e se tornarem necessidade em algum projeto e aí no geral é mais focado em libs/frameworks/tools pra casos específicos.

**Comment by felipefialho** [comment]
Todos os projetos que desenvolvo na empresa são em inglês, então é uma forma de treinar.

Durante o dia no trabalho estudo junto, mas normalmente relacionado a alguma coisa que preciso desenvolver e ainda não sei como fazer.

Por padrão costumo estudar 1h por dia à noite, de três a quatro vezes por semana, alternando entre Inglês (que também estou fazendo curso) e tecnologia.

**Comment by yanmagale** [comment]
Durante o trabalho, todos os projetos e documentação sao em inglês, já que temos um gringo no time e alguns parceiros são da Grécia. Durante o dia não constumo estudar, fora ouvir podcasts ou leitura de alguns artigos

Em casa, procuro organizar o tempo útil de forma que eu faça as aulas do inglês + a leitura de algo técnico. Geralmente procuro estudar sobre coisas fora o meu stack de trabalho.
